color-picker/build.gradle.kts

```diff
@@ -45,15 +42,17 @@ kotlin {
         }
         val jvmTest by getting
     }
+
+    tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
+        kotlinOptions.jvmTarget = "11"
+    }
 }
 
 android {
     namespace = "com.godaddy.common.colorpicker"
     compileSdk = 33
-    sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
```

Comment:
namespace from the line above has made the manifest useless. it was removed, so remove the reference